src/Symfony/Component/Form/ButtonBuilder.php

```diff
@@ -22,6 +25,11 @@
  */
 class ButtonBuilder implements \IteratorAggregate, FormBuilderInterface
 {
+    private const UNSUPPORTED_FORM_EVENTS = [
```

Comment:
This also define the supported form events, what about `UNSUPPORTED_SUPPORTED_FORM_EVENTS` instead?

Reply:
For me:

`FORM_EVENTS`= The 5 form events from the `FormEvents` class. 

`UNSUPPORTED_FORM_EVENTS` = Events from the `FormEvents` class that are not supported by buttons.

Reply:
Yes, but `self::UNSUPPORTED_FORM_EVENTS[$eventName]` = Events from the `FormEvents` class that are supported by buttons :)

Reply:
Alright, I see what you mean 😄 However, I still think the name applies to the whole const (what it represents) and not to its "values" only.

Reply:
My suggestion is to find a better name that makes it clearer. I could also name it `SUPPORTED_FORM_EVENTS` due to its values, so we've the chicken-and-egg situation here :)

Other names could be: `FORM_EVENTS_SUPPORT`, `BUTTON_EVENTS`, ...